### PR TITLE
Create Pin Screen

### LINKED ISSUE #3: Create Pin Screen
This screen consists of Enter Pin, Confirm Pin and Creating the Pin after Login.

### PR DESCRIPTION
**Pull request by SwaminathanS1215**
image

### CODE REVIEW

File: src/pages/createpin.css [hunk 1]
```diff
@@ -0,0 +1,145 @@
+h1, h2, h3, h4, h5, h6,p{
```
Comment (wissenshakti) [inline src/pages/createpin.css]: we can use tailwind.css

Comment (wissenshakti) [reply src/pages/createpin.css]: refrain from adding class to generic documents if it is not aplied through use classes

File: src/pages/createpin.css [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+    margin-bottom: 5px !important;
```
Comment (wissenshakti) [inline src/pages/createpin.css]: use specivity instead of !important

File: src/pages/createpin.css [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+    margin-top: 0 !important;
+  }
+  
+  .p-1{
```
Comment (wissenshakti) [inline src/pages/createpin.css]: tailwind or other css we can use for now it is fine . we can create a issue

File: src/pages/createpin.css [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+  
+  .--text-right{
+    text-align: right;
+  }
+  
+  .--font-bold{
+    font-weight: bold;
+  }
+  
+  ion-footer{
```
Comment (wissenshakti) [inline src/pages/createpin.css]: instead of tags uses class names

File: src/pages/createpin.css [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+    height: 82.67px;
```
Comment (wissenshakti) [inline src/pages/createpin.css]: refrain from using px . use rems for padding and other things. For this particuallare vh, %

Comment (wissenshakti) [reply src/pages/createpin.css]: could have user rem for this

File: src/pages/createpin.css [hunk 6]
```diff
@@ -0,0 +1,145 @@
...
+  }
+  
+ion-content{
+    --background:#020b3f !important;
```
Comment (wissenshakti) [inline src/pages/createpin.css]: we can use css variables and !important should be changed with specifivit

File: src/pages/createpin.css [hunk 7]
```diff
@@ -0,0 +1,145 @@
...
+ion-button.btn-orange{
+    --background: #ff5722;
+    --color: white;
+}
+
+ion-button.button-round{
+    --border-radius:7px;
+}
+
+ion-list.pin-flex {
```
Comment (wissenshakti) [inline src/pages/createpin.css]: remove this class , as it is not getting used anywher

File: src/pages/createpin.css [hunk 8]
```diff
@@ -0,0 +1,145 @@
...
+.pin-flex ion-item {
+    --background: transparent;
+    --padding-start: 0;
+    --inner-border-width: none;
+    --border-color: transparent;
+    display: flex;
+    --border-style: none;
+}
+
+.otp-ip input {
```
Comment (wissenshakti) [inline src/pages/createpin.css]: check and remove

File: src/pages/createpin.css [hunk 9]
```diff
@@ -0,0 +1,145 @@
...
+    padding: 10px !important;
+    font-size: 20px !important;
+    border: none;
+}
+
+.otp-ip input:focus-visible {
+    outline: none;
+}
+
+div[otpip="otpip"] {
```
Comment (wissenshakti) [inline src/pages/createpin.css]: rename the attribute is-input-otp

File: src/pages/createpin.tsx [hunk 10]
```diff
@@ -0,0 +1,64 @@
+import { useState, FC } from 'react';
```
Comment (wissenshakti) [inline src/pages/createpin.tsx]: File Name should be Pascal case , CreatPin.tsx

File: src/pages/createpin.tsx [hunk 11]
```diff
@@ -0,0 +1,64 @@
...
+import './createpin.css';
```
Comment (wissenshakti) [inline src/pages/createpin.tsx]: Pascal case on files

File: src/pages/createpin.tsx [hunk 12]
```diff
@@ -0,0 +1,64 @@
...
+import OtpInput from "react-otp-input";
+
+const CreatePin: FC = () => {
+
+    const [pin, setPin] = useState("")
+    const [confirmPin, setConfirmPin] = useState("")
+    const isInvalid = pin === "" || confirmPin === '' || pin !== confirmPin;
+
+
+    const handleChangePin = (e: any) => {
```
Comment (wissenshakti) [inline src/pages/createpin.tsx]: don't use any . Find out the type .

**Review by wissenshakti — CHANGES_REQUESTED** [review]

File: src/App.tsx [hunk 13]
```diff
@@ -11,6 +11,8 @@ import {
 import { IonReactRouter } from '@ionic/react-router';
 import { ellipse, square, triangle } from 'ionicons/icons';
 import Login from './pages/login';
+import CreatePin  from './pages/createpin';
```
Comment (sachinrath1986) [inline src/App.tsx]: file name should be CreatePin instead of createpin

File: src/pages/CreatePortfolioPin.css [hunk 14]
```diff
@@ -0,0 +1,129 @@
+/* Variables */
+:root {
```
Comment (wissenshakti) [inline src/pages/CreatePortfolioPin.css:2]: This needs to be moved to central place . Can create a issue for it

File: src/pages/CreatePortfolioPin.tsx [hunk 15]
```diff
@@ -0,0 +1,69 @@
+import { useState, FC } from 'react';
+import './CreatePortfolioPin.css';
```
Comment (wissenshakti) [inline src/pages/CreatePortfolioPin.tsx]: move it to last statement

**Review by wissenshakti — CHANGES_REQUESTED** [review]

**Review by wissenshakti — APPROVED** [review]

**Review by sachinrath1986 — APPROVED** [review]

**Pull request merged**